.github/copilot-instructions.md

```diff
@@ -0,0 +1,206 @@
+# Marc Littlemore's Personal Blog
+
+Marc Littlemore's personal blog is an 11ty (Eleventy) static site generator-based website with Tailwind CSS styling, deployed on Netlify. The site features blog posts, notes, games portfolio, talks, and integrations with external services like EmailOctopus, Goodreads, YouTube, and more.
+
+**Always reference these instructions first and fallback to search or bash commands only when you encounter unexpected information that does not match the info here.**
+
+## Working Effectively
+
+### Bootstrap and Build
+- **Node.js Version**: Use Node.js v20 (as specified in `.nvmrc`). Current system has v20.19.4 which is compatible.
```

Comment:
It should use the latest LTS version of Node.js. This could be updated in the .nvmrc file if needed.